swift-sdk/Internal/IterableAPIInternal.swift

```diff
@@ -710,6 +713,7 @@ final class IterableAPIInternal : NSObject, PushTrackerProtocol {
                                         dateProvider: dateProvider,
                                         retryInterval: config.inAppDisplayInterval)
         self.inAppManager = inAppManager
+        self.inboxManager = inAppManager
```

Comment:
Should this be inboxManager?

Reply:
inAppManager to the right is internal class which implements both inboxManager protocol and inAppManager protocol. In LHS we have external inBoxManager and inAppManager.

So inAppManager in LHS is different than inAppManager in RHS.